ui/components/multichain/create-btc-account/create-btc-account.tsx

```diff
@@ -1,61 +1,34 @@
 import React from 'react';
```

Comment:
We will need to remove those files when the Bitcoin Snap will start emitting the new suggested name!

Reply:
I'll add a todo